### PR TITLE
BH-68935 - Update CorporateUser documentation

### PR DESCRIPTION
**Pull request by cablke01**

### CODE REVIEW

File: source/includes/entityref/_corporateuser.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Represents an Internal user at your organization.
 | lastName | String (50) | CorporateUser’s last name. | | |
 | loginRestrictions | LoginRestrictions | A group of available login restrictions, including time, date, and IP address. | | |
 | massMailOptOut | Boolean | Indicates whether the CorporateUser opted out of mass mailings. | | |
-| masterUserID | Integer | Master User ID | X | X |
+| masterUserID | Integer | Master User ID. | X | X |
```
Comment (Allblues) [inline source/includes/entityref/_corporateuser.md:39]: Maybe change to
Master user id.

File: source/includes/entityref/_corporateuser.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ Represents an Internal user at your organization.
 | phone to phone3 | String (20) | Phone number of the CorporateUser. | | |
 | primaryDepartment | To-one association | Primary department of the CorporateUser. | | |
 | privateLabel | To-one association | Private Label associated to the CorporateUser. | X | |
```
Comment (Allblues) [inline source/includes/entityref/_corporateuser.md:50]: Make this one:
PrivateLabel

File: source/includes/entityref/_corporateuser.md [hunk 3]
```diff
@@ -48,12 +48,12 @@ Represents an Internal user at your organization.
...
-| privateLabels | To-many association | PrivateLabel | | |
+| privateLabels | To-many association | CorporateUser's PrivateLabels. | | |
 | reportToPerson | Person | Person to whom this CorporateUser reports. | | |
 | smsOptIn | Boolean | Indicates whether the CorporateUser has granted permission to be sent messages via SMS. | | |
 | status | String (100) | Status of the CorporateUser. | | |
 | taskAssignments | Task | Tasks that have been assigned to this CorporateUser, including those owned by the CorporateUser. | | |
 | timeZoneOffsetEST | Integer | Indicates the number of hours by which the CorporateUser’s time zone differs from Eastern Standard Time. For example, Pacific Standard Time is -3, three hours earlier than Eastern. | | |
 | userDateAdded | Timestamp | Date the record was added to the system. | X | X |
 | userType | To-one association | User Type associated to the CorporateUser. | X | |
```
Comment (Allblues) [inline source/includes/entityref/_corporateuser.md:58]: Let's go with:
User type

**Review by Allblues — CHANGES_REQUESTED** [review]

File: source/includes/entityref/_corporateuser.md (hunk 2 above)
Comment (cablke01) [reply source/includes/entityref/_corporateuser.md]: Updated

File: source/includes/entityref/_corporateuser.md (hunk 3 above)
Comment (cablke01) [reply source/includes/entityref/_corporateuser.md]: Updated

File: source/includes/entityref/_corporateuser.md (hunk 1 above)
Comment (cablke01) [reply source/includes/entityref/_corporateuser.md]: Updated

**Review by Allblues — APPROVED** [review]

**Pull request merged**